scripts/shared/dbRequestUtils.js

```diff
@@ -155,8 +155,12 @@ gpii.dbRequest.createResponseHandler = function (handleEnd, options, promise, er
                 promise.reject(fullErrorMsg);
             }
             else {
-                var value = handleEnd(responseString, options);
-                promise.resolve(value);
+                var handlingResult = handleEnd(responseString, options);
```

Comment:
Please update the doc comments for ResponseDataHandler to reflect this change in use

Reply:
No, I meant to update the actual definition of ResponseDataHandler at https://github.com/GPII/universal/blob/ef4bd8350cbca88705d37c17567259428a8d9926/scripts/shared/dbRequestUtils.js#L28 to indicate it now has a @return